### ISSUE TITLE
Text not shown properly, possibly a font problem

### ISSUE THREAD

**Issue by hveit** [mozilla/pdf.js#11151]
The attached PDF file does not show properly, when rendered with PDF.js. The console log shows fonts warnings.
I am not sure if this is a problem in the PDF file or in PDF.js.

Attach (recommended) or Link to PDF file here:
[937-Unfall.pdf](https://github.com/mozilla/pdf.js/files/3615972/937-Unfall.pdf)

Configuration:
- Web browser and its version: Google Chrome Version 77.0.3865.75 (Offizieller Build) (64-Bit)
- Operating system and its version: Win 7 Enterprise SP1
- PDF.js version: 1.10.100
- Is a browser extension:

Steps to reproduce the problem:
1. Upload and open the attached PDF file to PDF.js Demo on https://mozilla.github.io/pdf.js/web/viewer.html
2. Some text is out of place, too large, possibly due to wrong font used

What is the expected behavior? (add screenshot)
Google Chrome seems to render text correctly:
![ScreenshotChrome](https://user-images.githubusercontent.com/55389296/64954040-b9713e80-d884-11e9-97d1-fced8c694848.png)

As does Adobe Acrobat Reader DC:
![ScreenshotAcrobatReader](https://user-images.githubusercontent.com/55389296/64954056-c4c46a00-d884-11e9-9677-c3544f2bfe50.png)

What went wrong? (add screenshot)
Some text is out of place, too large:
![ScreenshotPDFJSOnlineDemo](https://user-images.githubusercontent.com/55389296/64954084-d3128600-d884-11e9-9532-e011a8e85591.png)

Browser console log shows font warnings:
![ScreenshotConsoleWarning](https://user-images.githubusercontent.com/55389296/64954227-3c929480-d885-11e9-9e28-346f2393032b.png)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by timvandermeij** [comment]
Closing since this is a duplicate of #7367 where not being able to load the fonts for AcroForms causes font substitution.

**Comment by THausherr** [comment]
I looked at field "4006" (Bearbeiter) and that one doesn't have an appearance stream. /NeedAppearances is set so the viewer should create it. /DA is "/QuickPDFF3ce08b65 10 Tf  0 0 0 rg" and the font "QuickPDFF3ce08b65" is available in the AcroForm default resources fully embedded ("Root/AcroForm/DR/Font/QuickPDFF3ce08b65/FontDescriptor/FontFile2") so no font substitution should be needed.

Thus I wonder whether PDF.js looks at the AcroForm default resources.

**Comment by hveit** [comment]
@THausherr thanks for your analysis.
So this issue doesn't seem to be a duplicate of issue #7367 after all and thus should be reopened?
Unfortunately I do not seem to be able to do this here...

**Comment by THausherr** [comment]
Yeah IMHO this should be reopened. Alternatively a look / explanation whether PDF.js uses the fonts in `AcroForm/DR/Font`.
@timvandermeij

**Comment by timvandermeij** [comment]
I have re-opened this since it seems to be a bit different, but I think the underlying cause is the same, namely that we don't do anything yet with the fonts from the `AcroForm` dictionary.

**Comment by hveit** [comment]
@timvandermeij thank you for clarifying.

**Comment by escapewindow** [comment]
This appears to be resolved if I set `renderInteractiveForms` to `true`.

**Comment by hveit** [comment]
> 
> 
> This appears to be resolved if I set `renderInteractiveForms` to `true`.

This works for me as well: The font problem seems to disappear, setting this option to true.
In this case all fields in my example PDF file are editable. This seems to be correct, since Adobe Acrobat reader shows it the same way.
Now my question is:
Is there some option/switch in PDF.js to show an AcroForm readonly, regardless of the readonly states of the fields within the PDF file? Something like a readonly mode for AcroForms? Or has something like this to be done always within the PDF file itself?

**Comment by timvandermeij** [comment]
Closing since this is fixed now with and without forms rendering, likely after #12828.
